lib/TableUtils.lua

```diff
@@ -26,7 +26,7 @@ local function getIterator(source)
 	end
 end
 
-function TableUtils.slice(tbl, first, last, step) --: (any[], number?, number?, number?) => any[]
+function TableUtils.slice(tbl, first, last, step) --: <T>(T[], number?, number?, number?) => T[]
```

Comment:
Are these types actually used for anything? Just thinking it's more to maintain and not sure what the value is.

Reply:
Given we've just both just misunderstood the contract of two separate methods I'd be inclined to say maintaining the types is valuable